packages/main-menu/src/MainManuItem.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ForwardedRef, forwardRef } from 'react'
+import { MainMenuItemProps } from './types'
+import { NavLink } from './MainMenuItemStyles'
+
+function MainMenuItem(
```

Comment:
Shouldn't this use `memo`?

Reply:
Also, don't we use arrow function everywhere?

Reply:
I'm not sure, but as far as I see, there are no arrow functions in other components. About `memo`, as I understand, it's not recommended to use until you face real performance issues
More info https://overreacted.io/before-you-memo/

Reply:
It's not the case here, we may not use `memo` when we know that component won't be re-rendered again, e.g. more or less static pages as lido-landing, but for ui-kit you don't know where it will be used, so we should use `memo` everywhere in ui-kit

Reply:
Btw if there are no `memo` in other components, we should create a ticket regarding it

Reply:
As you said, we don't know precisely where our components will be used, so I think we should provide the opportunity to choose. Developers can use `memo`  with our components in their projects if needed.

Reply:
Yep, thought about it for a bit, we probably really don't need it here